### PR TITLE
SEOImages: User Story 1463880, Part 32

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Updating image file names and references to improve SEO.

Fixes #1463880

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw This PR is ready for review. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019032000165/Report/

There are 2 image names identified by CATS as being too short. They should be handled at a later time as they are not part of the SEO image without hyphen work.

File: docs/framework/winforms/advanced/how-to-apply-gamma-correction-to-a-gradient.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ You can enable gamma correction for a linear gradient brush by setting the brush
   
  The following illustration shows the two filled rectangles. The top rectangle, which does not have gamma correction, appears dark in the middle. The bottom rectangle, which has gamma correction, appears to have more uniform intensity.  
   
- ![Gradient](./media/gammagradient1.png "gammagradient1")  
+ ![Diagram that shows two gradient-filled rectangles.](./media/how-to-apply-gamma-correction-to-a-gradient/two-rectangles-gamma-gradient.png)
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-apply-gamma-correction-to-a-gradient.md]:
```suggestion
 ![Two gradient-filled rectangles, with and without gamma correction.](./media/how-to-apply-gamma-correction-to-a-gradient/two-rectangles-gamma-gradient.png)  
```

File: docs/framework/wpf/advanced/hosting-win32-content-in-wpf.md [hunk 2]
```diff
@@ -361,7 +361,7 @@ public partial class Window1 : Window {
 
 Which gives you:
 
-![WPF application screenshot](./media/interoparch09.PNG "InteropArch09")
+![Windows Presentation Foundation application dialog screenshot](./media/hosting-win32-content-in-wpf/windows-presentation-foundation-application.png)
```
Comment (mairaw) [inline docs/framework/wpf/advanced/hosting-win32-content-in-wpf.md]:
```suggestion
![Screenshot of the WPF app running.](./media/hosting-win32-content-in-wpf/windows-presentation-foundation-application.png)
```

File: docs/framework/wpf/advanced/walkthrough-hosting-a-wpf-clock-in-win32.md [hunk 3]
```diff
@@ -24,11 +24,11 @@ This tutorial concentrates on the important steps of producing an interoperation
 
 The following graphic shows the intended end product of this tutorial:
 
-![Date and Time Properties dialog box](./media/interoparch06.PNG "InteropArch06")
+![Screenshot that shows the Date and Time Properties dialog box.](./media/walkthrough-hosting-a-wpf-clock-in-win32/date-time-properties-dialog.png)
 
 You can recreate this dialog by creating C++ [!INCLUDE[TLA2#tla_win32](../../../../includes/tla2sharptla-win32-md.md)] project in [!INCLUDE[TLA#tla_visualstu](../../../../includes/tlasharptla-visualstu-md.md)], and using the dialog editor to create the following:
```
Comment (mairaw) [inline docs/framework/wpf/advanced/walkthrough-hosting-a-wpf-clock-in-win32.md]:
```suggestion
You can recreate this dialog by creating a C++ Win32 project in Visual Studio, and using the dialog editor to create the following:
```

File: docs/framework/wpf/data/how-to-bind-the-properties-of-two-controls.md [hunk 4]
```diff
@@ -17,8 +17,7 @@ This example shows how to bind the property of one instantiated control to that
   
  When this example is rendered it looks like the following:  
   
- ![A canvas with a green background](./media/databindingbindingdpssample.PNG "DataBindingBindingDPsSample")  
-  
+ ![Screenshot showing a canvas with a green background.](./media/how-to-bind-the-properties-of-two-controls/data-binding-bind-background-canvas.png)
```
Comment (mairaw) [inline docs/framework/wpf/data/how-to-bind-the-properties-of-two-controls.md]:
```suggestion
![Screenshot showing a combo box with the value green selected and a green square.](./media/how-to-bind-the-properties-of-two-controls/data-binding-bind-background-canvas.png)

```

File: docs/framework/wpf/data/how-to-bind-the-properties-of-two-controls.md [hunk 5]
```diff
@@ -17,8 +17,7 @@ This example shows how to bind the property of one instantiated control to that
   
  When this example is rendered it looks like the following:  
   
- ![A canvas with a green background](./media/databindingbindingdpssample.PNG "DataBindingBindingDPsSample")  
-  
+ ![Screenshot showing a canvas with a green background.](./media/how-to-bind-the-properties-of-two-controls/data-binding-bind-background-canvas.png)    
  **Note** The binding target property (in this example, the <xref:System.Windows.Controls.Panel.Background%2A> property) must be a dependency property. For more information, see [Data Binding Overview](data-binding-overview.md).
```
Comment (mairaw) [inline docs/framework/wpf/data/how-to-bind-the-properties-of-two-controls.md]:
```suggestion
> [!NOTE]
> The binding target property (in this example, the <xref:System.Windows.Controls.Panel.Background%2A> property) must be a dependency property. For more information, see [Data Binding Overview](data-binding-overview.md).  
```

File: docs/framework/wpf/data/how-to-bind-to-xml-data-using-an-xmldataprovider-and-xpath-queries.md [hunk 6]
```diff
@@ -24,7 +24,7 @@ This example shows how to bind to [!INCLUDE[TLA#tla_xml](../../../../includes/tl
   
  The <xref:System.Windows.Controls.ListBox> will show the following items when this example is run. These are the *Title*s of all of the elements under *Books* with either a *Stock* value of "*out*" or a *Number* value of 3 or greater than or equals to 8. Notice that no *CD* items are returned because the <xref:System.Windows.Data.XmlDataProvider.XPath%2A> value set on the <xref:System.Windows.Data.XmlDataProvider> indicates that only the *Books* elements should be exposed (essentially setting a filter).  
   
- ![XPath Example](./media/xpathexample.PNG "XPathExample")  
+ ![Screenshow that shows an XPath example.](./media/how-to-bind-to-xml-data-using-an-xmldataprovider-and-xpath-queries/xpath-example-listbox-details.png)
```
Comment (mairaw) [inline docs/framework/wpf/data/how-to-bind-to-xml-data-using-an-xmldataprovider-and-xpath-queries.md]:
```suggestion
 ![Screenshot of the XPath example showing the title of four books.](./media/how-to-bind-to-xml-data-using-an-xmldataprovider-and-xpath-queries/xpath-example-listbox-details.png)  
```

File: docs/visual-basic/developing-apps/development-with-my/how-my-depends-on-project-type.md [hunk 7]
```diff
@@ -11,11 +11,11 @@ ms.assetid: c188b38e-bd9d-4121-9983-41ea6a94d28e
 ## My in Windows Applications and Web Sites  
  `My` exposes only objects that are useful in the current project type; it suppresses objects that are not applicable. For example, the following image shows the `My` object model in a Windows Forms project.  
   
- ![Shape of My in a Windows Forms application](../../../visual-basic/developing-apps/development-with-my/media/myinwinform.png "MyInWinForm")  
+ ![Diagram that shows the shape of My in a Windows Forms application.](./media/how-my-depends-on-project-type/my-shape-windows-forms-application.png)
```
Comment (mairaw) [inline docs/visual-basic/developing-apps/development-with-my/how-my-depends-on-project-type.md]:
```suggestion
 ![Diagram that shows the My object model in a Windows Forms application.](./media/how-my-depends-on-project-type/my-object-model-windows-forms.png)  
```

File: docs/visual-basic/developing-apps/development-with-my/how-my-depends-on-project-type.md [hunk 8]
```diff
@@ -11,11 +11,11 @@ ms.assetid: c188b38e-bd9d-4121-9983-41ea6a94d28e
 ## My in Windows Applications and Web Sites  
  `My` exposes only objects that are useful in the current project type; it suppresses objects that are not applicable. For example, the following image shows the `My` object model in a Windows Forms project.  
   
- ![Shape of My in a Windows Forms application](../../../visual-basic/developing-apps/development-with-my/media/myinwinform.png "MyInWinForm")  
+ ![Diagram that shows the shape of My in a Windows Forms application.](./media/how-my-depends-on-project-type/my-shape-windows-forms-application.png)  
   
  In a Web site project, `My` exposes objects that are relevant to a Web developer (such as the `My.Request` and `My.Response` objects) while suppressing objects that are not relevant (such as the `My.Forms` object). The following image shows the `My` object model in a Web site project:  
   
- ![Shape of My in a Web application](../../../visual-basic/developing-apps/development-with-my/media/myinweb.png "MyInWeb")  
+ ![Diagram that shows the shape of My in a Web application.](./media/how-my-depends-on-project-type/my-shape-web-application.png)
```
Comment (mairaw) [inline docs/visual-basic/developing-apps/development-with-my/how-my-depends-on-project-type.md]:
```suggestion
 ![Diagram that shows the My object model in a Web application.](./media/how-my-depends-on-project-type/my-object-model-web.png)  
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thank you @damabe. I've left some feedback for you before we can merge this.

**Comment by damabe** [comment]
@mairaw These changes have been made. Here's the new CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019041000041/Report/

**Review by mairaw — APPROVED** [review]
Thank you @damabe!

**Pull request merged**